src/components/Content/assets/IconPlaceholder.svg

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think we came to the conclusion that we need a different icon for ltr layouts. If we don't have one right now probably ask Anna Licia directly and create a task to change it.

Reply:
done